### PR TITLE
for #302: update list, claims, home, and base styles

### PR DESCRIPTION
**Pull request by groovecoder**
Some screenshots ...

codesy-home-scroll

codesy-home-ready

codesy-bids-list

codes-claim-paid

### CODE REVIEW

**Comment by jdungan** [comment]
Where did claim.user.get_big_gravatar get added?  It's didn't make it into this branch.

File: auctions/templates/bid_list.html [hunk 1]
```diff
@@ -6,28 +6,37 @@
...
             </tr>
         </thead>
         <tbody>
     {% for bid in bids %}
             {% for offer in bid.offers %}
             <tr>
+                <td>
+                    <span data-tooltip aria-haspopup="true" class="has-tip" data-disable-hover="false" title="{{ bid.created|date:"c" }}">{{ bid.created|date:"M j" }}</span></td>
                 <td><a href="{{bid.url}}">{{bid.issue.title}}</a></td>
-                <td>{{ bid.ask }}</td>
```
Comment (jdungan) [inline auctions/templates/bid_list.html]: Need to add back info on bids with asks.

File: auctions/templates/bid_list.html [hunk 2]
```diff
@@ -6,28 +6,38 @@
...
-                <td>{{ bid.offer }}</td>
-                <td>{% for fee in offer.fees %}
-                    <strong>{{ fee.fee_type }}</strong>: {{ fee.amount }}<br/>
-                    {% endfor %}
+                <td>
+                    <span
+                    {% if bid.ask_met %}
+                        data-tooltip aria-haspopup="true" class="has-tip" data-disable-hover="false" title="Your ask has been met"
+                    {% endif %}
+                    >{{ bid.ask }}</span>{% if bid.ask_met %} <i class="fi-check"></i>{% endif %}
```
Comment (groovecoder) [inline auctions/templates/bid_list.html]: The ``{{ bid.ask }}`` is here - along with a check if the ask has been met, and a mouse-over tooltip.

File: codesy/base/models.py [hunk 3]
```diff
@@ -20,9 +20,12 @@ class User(AbstractUser):
     stripe_account_token = models.CharField(max_length=100, blank=True)
     USERNAME_FIELD = 'username'
 
-    def get_gravatar_url(self):
+    def get_gravatar_url(self, size=40):
         email_hash = hashlib.md5(self.email).hexdigest()
-        return "//www.gravatar.com/avatar/{}?s=40".format(email_hash)
+        return "//www.gravatar.com/avatar/{}?s={}".format(email_hash, size)
+
+    def get_big_gravatar(self):
```
Comment (groovecoder) [inline codesy/base/models.py]: Here's the `get_big_gravatar` user method.

File: auctions/templates/bid_list.html (hunk 2 above)
Comment (jdungan) [inline auctions/templates/bid_list.html]: Found it. It was inside the offer for. I moved it out. Looks good now.

Comment (groovecoder) [inline auctions/templates/bid_list.html]: Not sure what you mean ... it's on the list view in the column before the offer column.
codesy-bid-ask

File: auctions/templates/bid_list.html [hunk 4]
```diff
@@ -27,7 +26,9 @@
                     {% if bid.ask_met %}
                         data-tooltip aria-haspopup="true" class="has-tip" data-disable-hover="false" title="Your ask has been met"
                     {% endif %}
-                    >{{ bid.ask }}</span>{% if bid.ask_met %} <i class="fi-check"></i>{% endif %}</td>
+                    >{{ bid.ask }}</span>{% if bid.ask_met %} <i class="fi-check"></i>{% endif %}
+                </td>
+            {% for offer in bid.offers %}
```
Comment (groovecoder) [inline auctions/templates/bid_list.html]: Should loop inside the `<td>` so we don't draw multiple columns in the outer table.

**Comment by jdungan** [comment]
I'm getting this error on three tests.  Here's the last one:

======================================================================
ERROR: test_get_gravatar_url_anon_user (codesy.tests.view_tests.HomeTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/owner/projects/codesy/codesy/codesy/tests/view_tests.py", line 55, in test_get_gravatar_url_anon_user
    url = self.view.get_gravatar_url()
AttributeError: 'Home' object has no attribute 'get_gravatar_url'

----------------------------------------------------------------------

**Comment by groovecoder** [comment]
Fixed, except for codecov noise. 😢

**Pull request merged**